1.11.2 Release Notes

anithamuthu79 opened the pull request:
NOTE:
The following commit message is incorrect and should be "Removed CoreOS 1688.5.3 since it is not supported for this release."
<img width="940" alt="screen shot 2018-05-17 at 10 41 34 am" src="https://user-images.githubusercontent.com/36425832/40194340-ece67ffa-59be-11e8-871d-e2003bcaeff8.png">

judithpatudith commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,124 @@
+Metronome 0.4.2 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.2).
+
+
+# <a name="issues-fixed"></a>Issues Fixed in DC/OS 1.11.2
```
HTML in this heading is unnecessary (left over from previous bad practices). Pleas just use markdown: `# Issues Fixed in DC/OS 1.11.2`

judithpatudith commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,124 @@
+- DCOS_OSS-2378/SOAK-88 - DC/OS Net: Improved stability of distribution protocol over TLS. [enterprise type="inline" size="small" /]
+
+
+# <a name="notable-changes"></a>Notable Changes in DC/OS 1.11.2
```
HTML in this heading is unnecessary (left over from previous bad practices). Please just use markdown: `# Notable Changes in DC/OS 1.11.2`

judithpatudith commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,124 @@
+2. Implemented region awareness support for SDK based services are implemented in this release.
+
+
+# <a name="known-issue"></a>Known Issue in DC/OS 1.11.2
```
HTML in this heading is unnecessary (left over from previous bad practices). And issues should be plural. Please just use markdown: `# Known Issues in DC/OS 1.11.2`

judithpatudith commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,124 @@
+- You can now select a DC/OS data service version from a dropdown menu in the DC/OS UI.
+- Improved scalability for all DC/OS data services.
+
+## <a name="known-issues"></a>Known Issues
```
There are two Known Issues sections in this doc. Please consolidate them so that there's only one section (the top one should satay and this one should go).

judithpatudith commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,124 @@
+- DCOS-34465 - Updated documentation on using multiple methods to SSH to a master node. 
+- DCOS_OSS-2132 - Added a new journal reader function `Follow()` to track the `journald` files rotation.
+- DCOS_OSS-2360 - Ordered numeric characters for Metric names in dcos-metrics Prometheus endpoint.
+- DCOS_OSS-2378/SOAK-88 - DC/OS Net: Improved stability of distribution protocol over TLS. [enterprise type="inline" size="small" /]
```
This is a soak issue so it might not be necessary to list in the public release notes. Please check.

urbanserj replied:
I don't know what we should mention in our release notes and what we shouldn't. However, I'm fine if you just keep "DCOS_OSS-2378".

anithamuthu79 replied:
@urbanserj I'll remove SOAK-88 and retain DCOS_OSS-2378. Thank you for your input. 
FYI:
Release Notes are documented to capture the following information:

-       What new features or functionalities have be implemented?

-       What improvements have been pushed out to existing features?

-       What bugs have been fixed?

-       What operation updates are done?

-       What known issues impact this release?

judithpatudith commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,124 @@
+
+# <a name="notable-changes"></a>Notable Changes in DC/OS 1.11.2
+
+- Updated to [Mesos 1.5.1](https://github.com/mesosphere/mesos/blob/b2eeb11ede805a7830cd6fb796d0b21a647aba04/CHANGELOG).
```
This conflicts with the info in line 17. Check which is correct, make sure line 17 reflects the correct version, and remove this line.

rukletsov replied:
https://github.com/mesosphere/mesos/commit/27d91e1fe46f09b2c74f2dc4efe4f58ae59ae0a8 looks correct

judithpatudith commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,124 @@
+# <a name="notable-changes"></a>Notable Changes in DC/OS 1.11.2
+
+- Updated to [Mesos 1.5.1](https://github.com/mesosphere/mesos/blob/b2eeb11ede805a7830cd6fb796d0b21a647aba04/CHANGELOG).
+- Updated to [Marathon 1.6.392](https://github.com/mesosphere/marathon/releases).
```
This line is redundant with line 18.

judithpatudith commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,124 @@
+
+- Updated to [Mesos 1.5.1](https://github.com/mesosphere/mesos/blob/b2eeb11ede805a7830cd6fb796d0b21a647aba04/CHANGELOG).
+- Updated to [Marathon 1.6.392](https://github.com/mesosphere/marathon/releases).
+- Updated to [Metronome 0.4.2](https://github.com/dcos/metronome/releases/tag/v0.4.2).
```
This line is redundant with line 19.

judithpatudith commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,124 @@
+- DCOS-29122 - Upgraded multi-region SDK package to Universe services.
+- DCOS-29634 - Implement region validation and config-sniffing in SDK.
+- DCOS_OSS-2377 - Agent operator API: Displayed resource provider resources in GET_RESOURCE_PROVIDER response.
+- DCOS_OSS-2688 - Supported secrets in Metronome.
```
Enterprise tag here

judithpatudith commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,124 @@
+**Note:** 
+1. Previous 1.11 point releases are not supported for CoreOS 1688.5.3.
+
+2. Implemented region awareness support for SDK based services are implemented in this release.
```
This might be enterprise-only?

anithamuthu79 replied:
Who is the point of contact to check/confirm?

judithpatudith replied:
Probably @somikbehera since he's the PM for Hybrid Cloud?

anithamuthu79 replied:
@somikbehera can you confirm, if "Implemented region awareness support for SDK based services are implemented in this release" requires an Enterprise tag?

anithamuthu79 replied:
@gabrielhartmann has confirmed in this PR, that we must use an ENT tag on "Implemented region awareness support for SDK based services are implemented in this release."

Added ENT tag.

judithpatudith commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,124 @@
+
+Provide feedback on the new features and services at: [support.mesosphere.com](https://support.mesosphere.com).
+
+<a name="new-features"></a>
```
I think this is a breadcrumb left over from previous bad practices (building tables of contents into each doc). I think it should be removed.

rukletsov commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,124 @@
+
+DC/OS 1.11.2 includes the following:
+
+Apache Mesos 1.5.x [change log](https://github.com/mesosphere/mesos/blob/27d91e1fe46f09b2c74f2dc4efe4f58ae59ae0a8/CHANGELOG).
```
DC/OS 1.11.2 does not include _any_ Mesos changes afaik. Do we still want to call Mesos changelog here?

rukletsov replied:
I lied, it indeed includes most of 1.5.x changes!

anithamuthu79 commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,119 @@
+
+- Apache Mesos 1.5.x [change log](https://github.com/mesosphere/mesos/blob/27d91e1fe46f09b2c74f2dc4efe4f58ae59ae0a8/CHANGELOG).
+- Marathon 1.6.392 [change log](https://github.com/mesosphere/marathon/releases).
+- Metronome 0.4.2 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.2).
```
@orsenthil Thanks for this input.

anithamuthu79 replied:
@orsenthil based on @jgehrcke https://github.com/dcos/dcos/blob/942262091fe07e540525763c4173886519ce3472/CHANGES.md#dcos-1112
Is there a change in the Metronome version or Metronome 0.4.2 is correct?
Can you confirm this?

orsenthil replied:
We can confirm this now.  The Marathon version that is being shipped is 0.4.2 - It got updated only yesterday.

anithamuthu79 replied:
Thank you @orsenthil

anithamuthu79 commented:
@gpaul could you review the changes made and let me know your feedback.

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+
+# Issues Fixed in DC/OS 1.11.2
+
+- DCOS-14199 - Fixed an issue that prevented DC/OS from recovering when available disk space was low.
```
This suggests that DC/OS now "recovers" when "disk space is low".

That is not true.

First of all, "Low disk space" is too unspecific. There black (ENOSPC) and white (disk space available).

Upon ENOSPC we know that DC/OS actually does not play nicely, irrespective of the actual change here.

This should be removed / replaced.

anithamuthu79 replied:
@adamtheturtle how do you want me to address this issue in documentation for 1.11.2 RN?

anithamuthu79 replied:
@adamtheturtle the following issue is re-worded.
- DCOS-14199 - Consolidated the Exhibitor bootstrapping shortcut by atomically reading and writing the ZooKeeper PID file.

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+
+- DCOS-14199 - Fixed an issue that prevented DC/OS from recovering when available disk space was low.
+- DCOS-21596 -  Added local user to LDAP group if local username matches LDAP username. [enterprise type="inline" size="small" /]
+- DCOS-22016 - Updated Prometheus documentation.
```
We do not want to list documentation updates. If we want to list them they should not be in "issues fixed".

anithamuthu79 replied:
Deleted DCOS-22016 entry.

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+- DCOS-14199 - Fixed an issue that prevented DC/OS from recovering when available disk space was low.
+- DCOS-21596 -  Added local user to LDAP group if local username matches LDAP username. [enterprise type="inline" size="small" /]
+- DCOS-22016 - Updated Prometheus documentation.
+- DCOS-22128 - Supported pods with volumes but no volume mounts. [enterprise type="inline" size="small" /]
```
This is a UI change and the current wording does not suggest so.

anithamuthu79 replied:
@Poltergeist can you help to reword the actual change for DCOS-22128?

Poltergeist replied:
How about `DCOS-22128 - Support pods with volumes but no volume mounts in the service ui.`? @jgehrcke @anithamuthu79

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+- DCOS-21596 -  Added local user to LDAP group if local username matches LDAP username. [enterprise type="inline" size="small" /]
+- DCOS-22016 - Updated Prometheus documentation.
+- DCOS-22128 - Supported pods with volumes but no volume mounts. [enterprise type="inline" size="small" /]
+- DCOS-34465 - Updated documentation on using multiple methods to SSH to a master node. 
```
We do not want to list documentation updates. If we want to list them they should not be in "issues fixed".

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+- DCOS-22016 - Updated Prometheus documentation.
+- DCOS-22128 - Supported pods with volumes but no volume mounts. [enterprise type="inline" size="small" /]
+- DCOS-34465 - Updated documentation on using multiple methods to SSH to a master node. 
+- DCOS_OSS-2132 - Added a new journal reader function `Follow()` to track the `journald` files rotation.
```
This actually landed in 1.11.1.

anithamuthu79 replied:
Removed DCOS_OSS-2132 entry.

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+- DCOS-22128 - Supported pods with volumes but no volume mounts. [enterprise type="inline" size="small" /]
+- DCOS-34465 - Updated documentation on using multiple methods to SSH to a master node. 
+- DCOS_OSS-2132 - Added a new journal reader function `Follow()` to track the `journald` files rotation.
+- DCOS_OSS-2360 - Ordered numeric characters for Metric names in dcos-metrics Prometheus endpoint.
```
This wording does not explain the change (how I understand it).

anithamuthu79 replied:
@philipnrmn can you reword this change, so I can document the actual change?

jgehrcke replied:
I think I understood the issue now and provided an alternative in the engineering changelog documents linked here: https://github.com/mesosphere/dcos-docs-site/pull/991#issuecomment-389590581

anithamuthu79 replied:
@Poltergeist reworded this change as:
DCOS_OSS-2360 - DC/OS Metrics: metric names are sanitized for better compatibility with Prometheus.

anithamuthu79 replied:
Cool, thanks!

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+
+# Notable Changes in DC/OS 1.11.2
+
+- DCOS-22021 - Removed IAM database migration from Zookeeper to Cockroach. [enterprise type="inline" size="small" /]
```
This is an internal change.

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+# Notable Changes in DC/OS 1.11.2
+
+- DCOS-22021 - Removed IAM database migration from Zookeeper to Cockroach. [enterprise type="inline" size="small" /]
+- DCOS-22041 - Removed policyquery cache race condition in Bouncer. [enterprise type="inline" size="small" /]
```
the user-facing change is that the race condition was removed in Admin Router.

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+
+- DCOS-22021 - Removed IAM database migration from Zookeeper to Cockroach. [enterprise type="inline" size="small" /]
+- DCOS-22041 - Removed policyquery cache race condition in Bouncer. [enterprise type="inline" size="small" /]
+- DCOS-22296 - Added the lvm2 command-line utilities (e.g. lvcreate, vgs, etc.) for csilvm CSI plugin. [enterprise type="inline" size="small" /]
```
this is an internal change

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+- DCOS-22021 - Removed IAM database migration from Zookeeper to Cockroach. [enterprise type="inline" size="small" /]
+- DCOS-22041 - Removed policyquery cache race condition in Bouncer. [enterprise type="inline" size="small" /]
+- DCOS-22296 - Added the lvm2 command-line utilities (e.g. lvcreate, vgs, etc.) for csilvm CSI plugin. [enterprise type="inline" size="small" /]
+- DCOS-22308 - Bumped CockroachDB to v1.1.8. [enterprise type="inline" size="small" /]
```
this is an internal change

anithamuthu79 replied:
Removed DCOS-22308 entry.

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+- DCOS-22041 - Removed policyquery cache race condition in Bouncer. [enterprise type="inline" size="small" /]
+- DCOS-22296 - Added the lvm2 command-line utilities (e.g. lvcreate, vgs, etc.) for csilvm CSI plugin. [enterprise type="inline" size="small" /]
+- DCOS-22308 - Bumped CockroachDB to v1.1.8. [enterprise type="inline" size="small" /]
+- DCOS-22326 - Disabled the insecure 3DES bulk encryption algorithm and TLS 1.1 protocol by default for Master Admin Router. [enterprise type="inline" size="small" /]
```
this must go into the security section

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+- DCOS-22296 - Added the lvm2 command-line utilities (e.g. lvcreate, vgs, etc.) for csilvm CSI plugin. [enterprise type="inline" size="small" /]
+- DCOS-22308 - Bumped CockroachDB to v1.1.8. [enterprise type="inline" size="small" /]
+- DCOS-22326 - Disabled the insecure 3DES bulk encryption algorithm and TLS 1.1 protocol by default for Master Admin Router. [enterprise type="inline" size="small" /]
+- DCOS-22439 - Promoted metrics documentation to staging. 
```
We do not want to list documentation updates. If we want to list them they should not be in "notable changes" but more in a "documentation updates" section or so.

anithamuthu79 replied:
Removed DCOS-22308 entry.

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+- DCOS-22308 - Bumped CockroachDB to v1.1.8. [enterprise type="inline" size="small" /]
+- DCOS-22326 - Disabled the insecure 3DES bulk encryption algorithm and TLS 1.1 protocol by default for Master Admin Router. [enterprise type="inline" size="small" /]
+- DCOS-22439 - Promoted metrics documentation to staging. 
+- DCOS-29122 - Upgraded multi-region SDK package to Universe services.
```
Is that a UI change?

jgehrcke replied:
I cannot understand what the user-facing change is by reading the ticket.

I understand that no change landed in DC/OS or DC/OS Enterprise (I looked at the commits).

I think this should be removed until we understand what this means

anithamuthu79 replied:
@imaxxs and Sukesh have worked on this issue.
@imaxxs let me know, if this issue needs a better description or should be removed from the current documentation.

imaxxs replied:
@anithamuthu79 @jgehrcke This describes that now all the SDK based packages on Universe support multi-region support. This is not UI but SDK + Each DS change. So yes, this should be documented. We can possibly change the text to - "Support for launching services in a remote region"

anithamuthu79 replied:
Thanks @imaxxs

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+- DCOS-22326 - Disabled the insecure 3DES bulk encryption algorithm and TLS 1.1 protocol by default for Master Admin Router. [enterprise type="inline" size="small" /]
+- DCOS-22439 - Promoted metrics documentation to staging. 
+- DCOS-29122 - Upgraded multi-region SDK package to Universe services.
+- DCOS-29634 - Implement region validation and config-sniffing in SDK.
```
Is this tied to a specific SDK release?

I don't think this is part of the 1.11.2 DC/OS release. If it is -- who can explain why that is?

anithamuthu79 replied:
@jgehrcke I don't think you will have the correct answer, as the bug reporter is not a Mesosphere employee anymore.
Let me know, if you can reword this change or should I remove this entry from the current documentation.

anithamuthu79 replied:
@orsenthil @imaxxs can you help in answering this question for @jgehrcke?

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+- DCOS-29122 - Upgraded multi-region SDK package to Universe services.
+- DCOS-29634 - Implement region validation and config-sniffing in SDK.
+- DCOS_OSS-2377 - Agent operator API: Displayed resource provider resources in GET_RESOURCE_PROVIDER response.
+- DCOS_OSS-2688 - Supported secrets in Metronome. [enterprise type="inline" size="small" /]
```
This is already covered in the (linked) changelog for Metronome 0.4.2.

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+
+# Known Issue in DC/OS 1.11.2
+
+- DCOS-20568 - Fixed dcos-diagnostics master service account permission issue.
```
Yes, this has been fixed in this release :-) -- it is not a known issue in this release.

anithamuthu79 replied:
Removed DCOS-20568 entry.

jgehrcke commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,118 @@
+- DCOS-29634 - Implement region validation and config-sniffing in SDK.
+- DCOS_OSS-2377 - Agent operator API: Displayed resource provider resources in GET_RESOURCE_PROVIDER response.
+- DCOS_OSS-2688 - Supported secrets in Metronome. [enterprise type="inline" size="small" /]
+- QUALITY-2006 - RHEL 7.4 with Docker EE 17.06.2 is supported.
```
Did we actually try this with the 1.11.2 release candidate? I cannot see this from the JIRA ticket.

About a month ago @gauripowale wrote in the JIRA ticket:

> I was able to qualify and confirm that DC/OS 1.11 works with RHEL 7.4 and Docker EE 17.06.2 

It is not clear which DC/OS 1.11 build was used for testing. If it was the latest at that time (April 21) then the current one is different from that.

I do not know if we should have this in the release notes if we are not 100 % certain that this works with the release.

anithamuthu79 replied:
@gauripowale @masonse can you share your confirmation on this change here. This will help to track all communications in one place.

gauripowale replied:
@jgehrcke is correct, I tested against `test/1.11` a couple weeks ago which was an older version of 1.11.2 than now. He brings up a good point, I'll re-test based on the latest 1.11.2 that is on `test/1.11`.

jgehrcke commented:
> I might not be able to provide a better proposal in time, but I am trying in parallel to have the changelog that lives side-by-side with the 1.11 code reflect reality.

We got that now.

@anithamuthu79 the following two documents is what the release notes should be based on:
* [Curated list of user-facing changes for DC/OS 1.11.2](https://github.com/dcos/dcos/blob/942262091fe07e540525763c4173886519ce3472/CHANGES.md#dcos-1112)
* [Curated list of user-facing changes for DC/OS Enterprise 1.11.2](https://github.com/mesosphere/dcos-enterprise/blob/3847d620896c2bd7f1d801444662e377ec3b467c/CHANGES.md#dcos-enterprise-1112)

These are based on the actual commits between 1.11.1 and 1.11.2 and I tried to keep the items comprehensive.

anithamuthu79 commented:
@jgehrcke @judithpatudith @orsenthil 

Can you also check if the Marathon change log is correct?
- Marathon 1.6.392 [change log](https://github.com/mesosphere/marathon/releases).

@jgehrcke pls clarify, if these changes should go into the current documentation for 1.11.2 RN?

The following entries are listed on 

https://github.com/mesosphere/dcos-enterprise/blob/3847d620896c2bd7f1d801444662e377ec3b467c/CHANGES.md#dcos-enterprise-1112

https://github.com/dcos/dcos/blob/942262091fe07e540525763c4173886519ce3472/CHANGES.md#dcos-1112

but not on https://jira.mesosphere.com/issues/?filter=28631

- DCOS-22133 - DC/OS IAM: Fixed a rare case where the database bootstrap transaction would not insert some data. 

- COPS-3195 - Mesos: Fixed an issue as of which the authentication token refresh would not be performed. 

- DC/OS UI: incorporated [multiple](https://github.com/dcos/dcos/pull/2799) fixes and improvements.
**QUESTION:** Is it a good practice to include the PR link as shown above in the RN documentation OR should it be documented as "- DC/OS UI: incorporated multiple fixes and improvements."

- DCOS-21958 - Admin Router on the DC/OS master nodes now does not support TLS 1.1 and the 3DES bulk encryption algorithm anymore by default. 

- MARATHON-8090 - Reverted the Marathon configuration change for GPU resources which was introduced with the 1.11.1 release. 

- DCOS-21611 - The IP detect script and fault domain detect script may now be changed with a config upgrade. 

- DCOS_OSS-2317 - Consolidated pkgpanda's package download method. 

- DCOS_OSS-2335 - Increased the Mesos executor re-registration timeout to consolidate an agent failover scenario.

gauripowale commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,106 @@
+- QUALITY-2006 - RHEL 7.4 with Docker EE 17.06.2 is supported.
+- QUALITY-2007 - RHEL 7.4 with Docker 17.12.1-ce is supported. 
+- QUALITY-2057 - CentOS 7.4 with Docker EE 17.06.2 is supported.
+- QUALITY-2060 - Certified DC/OS 1.11.2 with CoreOS 1688.5.3.
```
@anithamuthu79 Please remove QUALITY-2060 (DCOS-37144 now after the JIRA migration) from the 1.11.2 release notes. The networking fix for this did not make it into the 1.11.2 branch. This should also be removed from the compatibility matrix on the version policy page as well. The CoreOS 1688.5.3 fix will be in 1.11.3.

cc @orsenthil @jgehrcke @masonse

orsenthil commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,106 @@
+# Issues Fixed in DC/OS 1.11.2
+
+- DCOS-14199 - Consolidated the Exhibitor bootstrapping shortcut by atomically reading and writing the ZooKeeper PID file.
+- DCOS-20514 - Added licensing information to the diagnostics bundle.
```
`DCOS-20514 - Added licensing information to the diagnostics bundle.`

Is this new in **1.11.2** ? The JIRA says it was already introduced in **1.11.1**

orsenthil replied:
We should remove this line.

anithamuthu79 replied:
@orsenthil based on @jgehrcke suggestion, I've included DCOS-20514 in 1.11.2 RN documentation.

orsenthil commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,106 @@
+- DCOS-14199 - Consolidated the Exhibitor bootstrapping shortcut by atomically reading and writing the ZooKeeper PID file.
+- DCOS-20514 - Added licensing information to the diagnostics bundle.
+- DCOS-21596 - Added local user to LDAP group if local username matches LDAP username. [enterprise type="inline" size="small" /]
+- DCOS-22128 - Supported pods with volumes but no volume mounts. [enterprise type="inline" size="small" /]
```
`DCOS-22128 - Fixed a bug in the Service view of DC/OS UI when cluster has pods with not every container mounting a volume`

anithamuthu79 replied:
Rephrased it.

orsenthil commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,106 @@
+
+- DCOS-21958 - Admin Router on the DC/OS master nodes now does not support TLS 1.1 and the 3DES bulk encryption algorithm anymore by default.
+- DCOS-22326 - Disabled the insecure 3DES bulk encryption algorithm and TLS 1.1 protocol by default for Master Admin Router. [enterprise type="inline" size="small" /] 
+- DCOS-29122 - Support for launching services in a remote region.
```
I am requesting @benclarkwood  and @gabrielhartmann  to review these entries for DC/OS 1.11.2 changelog.

```
+- DCOS-29122 - Support for launching services in a remote region.
+- DCOS-29634 - Implement region validation and config-sniffing in SDK.
```

* Are these accurate descriptions for the feature supported in 1.11.2?
* Is this accomplished in 1.11.2?
* Should this be mentioned in changelog? (A noteworthy change that a customer is going to find it useful). 

I see an epic called (Region Aware Scheduling in Data Services https://jira.mesosphere.com/browse/DCOS-29023) still in In-progress, so I am unable to take call on stating these two entries as "features" supported by DC/OS 1.11.2 

TIA!

imaxxs replied:
@orsenthil EPIC https://jira.mesosphere.com/browse/DCOS-29575 was in progress because the documentation ticket is open. Sukesh should close this ticket. We have released this and you can check the service docs - https://docs.mesosphere.com/services/kafka/2.2.0-1.0.0/release-notes/ This feature is released after 1.11 release so to answer your questions - 

1. Are these accurate descriptions for the feature supported in 1.11.2? 
Yes; Check this release notes https://docs.mesosphere.com/services/kafka/2.2.0-1.0.0/release-notes/ 
2. Is this accomplished in 1.11.2? 
Yes
3. Should this be mentioned in changelog? (A noteworthy change that a customer is going to find it useful).
Yes

orsenthil replied:
Thank you! @imaxxs !

orsenthil commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,106 @@
+- DCOS-22326 - Disabled the insecure 3DES bulk encryption algorithm and TLS 1.1 protocol by default for Master Admin Router. [enterprise type="inline" size="small" /] 
+- DCOS-29122 - Support for launching services in a remote region.
+- DCOS-29634 - Implement region validation and config-sniffing in SDK.
+- DCOS_OSS-2377 - Agent operator API: Displayed resource provider resources in GET_RESOURCE_PROVIDER response.
```
This is covered by Mesos version change.  Mentioning it separately is not required.

anithamuthu79 replied:
Removed DCOS_OSS-2377 entry.

orsenthil commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,106 @@
+- DCOS-29122 - Support for launching services in a remote region.
+- DCOS-29634 - Implement region validation and config-sniffing in SDK.
+- DCOS_OSS-2377 - Agent operator API: Displayed resource provider resources in GET_RESOURCE_PROVIDER response.
+- QUALITY-2006 - RHEL 7.4 with Docker EE 17.06.2 is supported.
```
These will be approved by @gauripowale  as she tested this.

Also, this statement in the changelog.

> Previous 1.11 point releases are not supported for CoreOS 1688.5.3.

gauripowale replied:
I re-tested QUALITY-2006 (RHEL 7.4 with Docker EE 17.06.2), QUALITY-2007 (RHEL 7.4 with Docker 17.12.1-ce), and QUALITY-2057 (CentOS 7.4 with Docker EE 17.06.2) against 1.11.2, which all passed the integration tests and thus should remain in the Release Notes.

However, any reference to CoreOS 1688.5.3 should be removed from the RN as the support for this platform will land in 1.11.3.
cc @orsenthil @jgehrcke

orsenthil commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,106 @@
+**Note:** 
+1. Previous 1.11 point releases are not supported for CoreOS 1688.5.3.
+
+2. Implemented region awareness support for SDK based services are implemented in this release. [enterprise type="inline" size="small" /]
```
@benclarkwood  / @gabrielhartmann  - Please confirm if this is accurate.

I see an epic called (Region Aware Scheduling in Data Services https://jira.mesosphere.com/browse/DCOS-29023) still in In-progress, so I am unable to take call on stating these two entries as "features" supported by DC/OS 1.11.2

orsenthil replied:
This is answered by @imaxxs

orsenthil commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,106 @@
+
+2. Implemented region awareness support for SDK based services are implemented in this release. [enterprise type="inline" size="small" /]
+
+3. New Docker versions supported on RHEL 7.4 and CoreOS 1688.5.3. See [compatibility matrix](https://docs.mesosphere.com/version-policy/) for further information.
```
To be approved by @gauripowale .

gauripowale replied:
Re-tested RHEL 7.4 against 1.11.2, which passed all tests. Please include this in the RN & in the compatibility matrix. You can remove CoreOS 1688.5.3 from the release notes and the compatibility matrix as this fix for this will land in 1.11.3.

anithamuthu79 replied:
@gauripowale thank you for the confirmation. I've removed all CoreOS 1688.5.3 entries from 1.11.2 RN documentation.

judithpatudith commented:
Looks like all my comments have been addressed in one form or another.

anithamuthu79 commented:
@jgehrcke @orsenthil @judithpatudith could you help in identifying, if these entries require an ENT tag or not?

- DCOS-22133 - DC/OS IAM: Fixed a rare case where the database bootstrap transaction would not insert some data.

- COPS-3195 - Mesos: Fixed an issue as of which the authentication token refresh would not be performed.
- DC/OS UI: incorporated multiple fixes and improvements.
**QUESTION: Is it a good practice to include the PR link as shown above in the RN documentation OR should it be documented as "- DC/OS UI: incorporated multiple fixes and improvements."**

- DCOS-21958 - Admin Router on the DC/OS master nodes now does not support TLS 1.1 and the 3DES bulk encryption algorithm anymore by default.

- MARATHON-8090 - Reverted the Marathon configuration change for GPU resources which was introduced with the 1.11.1 release.

- DCOS-21611 - The IP detect script and fault domain detect script may now be changed with a config upgrade.

- DCOS_OSS-2317 - Consolidated pkgpanda's package download method.

- DCOS_OSS-2335 - Increased the Mesos executor re-registration timeout to consolidate an agent failover scenario.

orsenthil commented on pages/1.11/release-notes/1.11.2/index.md:
```diff
@@ -0,0 +1,109 @@
+
+# Notable Changes in DC/OS 1.11.2
+
+- DCOS-22326 - Disabled the insecure 3DES bulk encryption algorithm and TLS 1.1 protocol by default for Master Admin Router. [enterprise type="inline" size="small" /] 
```
This can be moved under Security enhancements.

SarahStegall commented:
*applause*

On Thu, May 17, 2018 at 3:25 PM, anithamuthu79 <notifications@github.com>
wrote:

> *@anithamuthu79* commented on this pull request.
> ------------------------------
>
> In pages/1.11/release-notes/1.11.2/index.md
> <https://github.com/mesosphere/dcos-docs-site/pull/991#discussion_r189119097>
> :
>
> > +- DCOS-20568 - Fixed diagnostics bundle creation bug as of insufficient service account permissions. [enterprise type="inline" size="small" /]
> +- DCOS-21596 - Added local user to LDAP group if local username matches LDAP username. [enterprise type="inline" size="small" /]
> +- DCOS-21611 - The IP detect script and fault domain detect script can be changed with a config upgrade.
> +- DCOS-22128 - Fixed an issue in the Service view of DC/OS UI, when cluster has pods with not every container mounting a volume [enterprise type="inline" size="small" /]
> +- DCOS-22041 - Admin Router: Fixed a race condition in the permission data cache. [enterprise type="inline" size="small" /]
> +- DCOS-22133 - DC/OS IAM: Fixed a rare case where the database bootstrap transaction would not insert some data. [enterprise type="inline" size="small" /]
> +- DCOS_OSS-2317 - Consolidated pkgpanda's package download method.
> +- DCOS_OSS-2335 - Increased the Mesos executor re-registration timeout to consolidate an agent failover scenario.
> +- DCOS_OSS-2360 - DC/OS Metrics: metric names are sanitized for better compatibility with Prometheus.
> +- DCOS_OSS-2378 - DC/OS Net: Improved stability of distribution protocol over TLS.
> +- DC/OS UI: Incorporated [multiple](https://github.com/dcos/dcos/pull/2799) fixes and improvements.
> +
> +
> +# Notable Changes in DC/OS 1.11.2
> +
> +- DCOS-22326 - Disabled the insecure 3DES bulk encryption algorithm and TLS 1.1 protocol by default for Master Admin Router. [enterprise type="inline" size="small" /]
>
> Moved DCOS-22326 to Security Enhancements section.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/dcos-docs-site/pull/991#discussion_r189119097>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/Aj2s_XfRdISrQQE4DuFu5PPgu3WQrTeVks5tzfjKgaJpZM4T5Eai>
> .
>



-- 
Sarah Stegall
Technical Writer, Mesosphere
